packages/hooks/README.md

```diff
@@ -0,0 +1,106 @@
+# Slack CLI Hooks
+
+The `@slack/hooks` package contains scripts that implement the contract between
+the [Slack CLI][cli] and [Bolt for JavaScript][bolt].
+
+## Overview
+
+This library enables inter-process communication between the Slack CLI and
+applications built with Bolt for JavaScript.
+
+When used together, the CLI delegates various tasks to the Bolt application by
+invoking processes ("hooks") and then making use of the responses provided by
+each hook's `stdout`.
+
+For a complete list of available hooks, read the [Supported Hooks][supported]
+section.
+
+## Requirements
+
+This package supports Node v18 and higher. It's highly recommended to use [the
+latest LTS version of Node][node].
+
+An updated version of the Slack CLI is also encouraged while using this package.
+
+## Installation
```

Comment:
Including this to match other Node packages, but open to removing it. I'm thinking any errors encountered are expected in early versions, but not thinking very much